utility/mirrormanager2.spec

```diff
@@ -255,28 +251,28 @@ install -m 644 utility/mirrormanager2.cfg.sample \
 
 # Install crawler logrotate definition
 install -m 644 utility/mm2_crawler.logrotate \
-    $RPM_BUILD_ROOT/%{_sysconfdir}/logrotate.d/mm2_crawler
+    $RPM_BUILD_ROOT/%{_sysconfdir}/logrotate.d/mirrormanager2-crawler
```

Comment:
All the renames from mm2- to mirrormanager2- are also unexpected.

Reply:
The renaming was to fix the rpmlint error `incoherent-logrotate-file`. Seems that rpmlint insist on the logrotate file to be of the same name as the package name: https://fedoraproject.org/wiki/Common_Rpmlint_issues#incoherent-logrotate-file

Reply:
But this will break existing installations. We should not do that.

Reply:
Moving `incoherent-logrotate-file` error to rpmlintrc.